bridges/ethElrond/constants.go

```diff
@@ -6,3 +6,14 @@ import "time"
 const InvalidActionID = uint64(0)
 
 const durationLimit = time.Duration(time.Second)
+
+type ClientStatus int
+
+const (
+	Available ClientStatus = 0
+	Unavailable
+)
+
+func (cs ClientStatus) String() string {
+	return []string{"Available", "Unavailable"}[cs]
```

Comment:
let's use a switch. Otherwise I can easily panic the code with something like:
```GO
a := ClientStatus(2)
_ = a.String()

```

Reply:
done